From e13f71514b0b6ca3b3e15cb8013a8db1e47d3ea9 Mon Sep 17 00:00:00 2001
From: Chris Coulson <chris.coulson@canonical.com>
Date: Wed, 11 Mar 2020 16:46:41 +0100
Subject: ubuntu: Update the linux boot protocol version check.

The EFI implementation of grub_cmd_linux makes use of xloadflags which was
introduced in to version 2.12 of the kernel's boot protocol, so update the
check accordingly.
---
 grub-core/loader/i386/efi/linux.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: grub2-2.02~beta2/grub-core/loader/i386/efi/linux.c
===================================================================
--- grub2-2.02~beta2.orig/grub-core/loader/i386/efi/linux.c
+++ grub2-2.02~beta2/grub-core/loader/i386/efi/linux.c
@@ -226,7 +226,7 @@ grub_cmd_linux (grub_command_t cmd __att
       goto fail;
     }
 
-  if (lh.version < grub_cpu_to_le16 (0x020b))
+  if (lh.version < grub_cpu_to_le16 (0x020c))
     {
       grub_error (GRUB_ERR_BAD_OS, N_("kernel too old"));
       goto fail;
